processing/src/main/java/io/druid/segment/serde/ComplexColumnPartSerde.java

```diff
@@ -59,21 +58,110 @@ public String getTypeName()
     return typeName;
   }
 
-  @Override
-  public long numBytes()
+  public static SerializerBuilder serializerBuilder()
```

Comment:
here and in other places, SerializerBuilder is really building [Complex]ColumnPartSerde and not "serializer" of the ColumnPartSerde. can we rename it to be ColumnPartSerdeBuilder instead?